p1

```diff
@@ -0,0 +1,2 @@
+Identity added: /home/mikuhnt/.ssh/id_rsa (/home/mikuhnt/.ssh/id_rsa)
```

Comment:
I am guessing this should not be here :smile:

Reply:
damn .. thx